components/previous-apprentices/previous-apprentices.module.scss

```diff
@@ -3,6 +3,13 @@
 @use '../../styles/fonts' as *;
 
 .previous-apprentices {
+  background: url('../../public/sparkbox-logo.svg') no-repeat;
+  background-size: 40rem;
+  background-position: 68rem;
```

Comment:
Nice work Alice, I think we just need to make some adjustments to the background position. The larger the screen gets, the farther to the left the logo moves. At a 4K screen width `3840px`, the logo is almost all the way on the left.

We can use `calc` to position it horizontally from the center, so that it stays in a fixed position regardless of how large the screen gets. I eyeballed these values in the devtools, but we'll probably want to adjust it and get J's feedback.

```css
background-position-x: calc(50% + 38rem);
background-position-y: 16rem;
```

Also, I think we only need to apply the bg image above a particular screen width. J's designs don't appear to show it on smaller screen sizes.